docs/Windows_Installation_Instructions.md

```diff
@@ -0,0 +1,176 @@
+# Instructions for Windows
+
+This document describes how to configure and start up a running version of MMS 4.
+
+Versions Used to Test these instructions
+* [MMS v4.0.0-a2](https://github.com/Open-MBEE/mms/tree/4.0.0-a2)
```

Comment:
There is a newer image now `[MMS v4.0.0-b2](https://github.com/Open-MBEE/mms/tree/4.0.0-b2)`